bucket/k9s.json

```diff
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://github.com/derailed/k9s",
+    "license": "Apache-2.0",
+    "description": "Kubernetes CLI To Manage Your Clusters In Style!",
+    "version": "0.7.2",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/derailed/k9s/releases/download/0.7.2/k9s_0.7.2_Windows_x86_64.tar.gz",
+            "hash": "sha256:df8fb1b41d96abef5be473f62c4f4a0ae533f8ea319992a5fc7d63cfe18828df"
```

Comment:
Remove sha256 prefix. Sha256 is default.

To automatically fix both run `.\bin\checkver.ps1 k9s -f` from repository root